tests/admin_views/tests.py

```diff
@@ -3755,6 +3771,21 @@ def test_recentactions_link(self):
         should_contain = """<a href="%s">%s</a>""" % (escape(link), escape(self.pk))
         self.assertContains(response, should_contain)
 
+    def test_recentactions_changed(self):
+        response = self.client.get(reverse("admin:index"))
+        should_contain = '<span class="visually-hidden">Changed:'
+        self.assertContains(response, should_contain)
+
+    def test_recentactions_delete(self):
+        response = self.client.get(reverse("admin:index"))
+        should_contain = '<span class="visually-hidden">Deleted:'
+        self.assertContains(response, should_contain)
+
+    def test_recentactions_added(self):
+        response = self.client.get(reverse("admin:index"))
+        should_contain = '<span class="visually-hidden">Added:'
+        self.assertContains(response, should_contain)
```

Comment:
I would move it to the single test method, e.g.
```suggestion
    def test_recentactions_description(self):
        response = self.client.get(reverse("admin:index"))
        for operation in ["Added", "Changed", "Deleted"]:
            with self.subTest(operation):
                self.assertContains(response, f'<span class="visually-hidden">{operation}:')
```